Sources/StreamChat/Repositories/PollsRepository.swift

```diff
@@ -288,6 +288,11 @@ class PollsRepository {
             )
         }
     }
+    
+    struct VotePaginationResponse {
```

Comment:
Why not use `PollVoteListResponse`?

Reply:
Yes seems like we could just reuse `PollVoteListResponse` 🤔

Reply:
I added a separate one since `PollVoteListResponse` is API payload and this returns what DB has after the whole conversion cycle.

Reply:
Ah I see 👍